run_evals_from_checkpoints.py

```diff
@@ -0,0 +1,380 @@
+import glob
```

Comment:
(very minor nit): this seems unused, the globs are all performed as `path.glob()` rather than using the glob import